internal/server/ranking/ranking.go

```diff
@@ -68,6 +68,10 @@ var StatsRanking = map[RankKey]int{
 	{"NASA_WetBulbComputation_Aggregation", "NASA_Mean_HadGEM2-AO", "*"}: 0, // Wet bulb year aggregation
 	{"NASA_WetBulbComputation_Aggregation", "*", "*"}:                    1, // Wet bulb year aggregation
 	{"NASA_WetBulbComputation", "NASA_Mean_HadGEM2-AO", "*"}:             2, // Wet bulb
+
+	// Prefer FBI Hate Crime Publications over Data Commons Aggregate
```

Comment:
Could you add note website links that can validate this change upon submission to autopush?

Reply:
Done. Have added validation links at country and state level.

Reply:
Have made a fix.
The issue originated due to the new config folder created in [PR 1392](https://github.com/datacommonsorg/website/pull/1392) of the website repo.
Have made the fix in this PR.